daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/dynatrace.md

```diff
@@ -0,0 +1,83 @@
+---
+type: docs
+title: "How-To: Set up Dynatrace OneAgent for distributed tracing"
+linkTitle: "Dynatrace"
+weight: 1000
+description: "Set up Dynatrace OneAgent for distributed tracing and deep code-level insights in Kubernetes"
+---
+
+[Dynatrace](https://www.dynatrace.com) OneAgent is responsible for collecting all telemetry within your monitored environment. 
+
+It automatically instruments your applications built with Dapr without the need for further configuration. 
+
+![Distributed Trace](/images/dt-waterfall.png)
+
+![Codelevel-Trace](/images/dt-ppath.png)
+
+## Installation
```

Comment:
I'm not seeing any guidance regarding the required Dapr configuration as in the guidance for other backends (e.g. where to point the Dapr traces). Is this missing? if not needed due to the OneAgent I think it still needs to be explicitly mentioned as it may be confusing to the reader